config/prow/cluster/hook_github_app_service.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
in most of the place we just use the naming convention used for the file name is like `<name>_<type>.yaml` e.g: hook_service.yaml, hence can we rename this file as `hook2_service.yaml`

Reply:
Done.